### PR TITLE
Toggle tile cache final API

### PR DESCRIPTION
**Pull request by alexcristici**

### CODE REVIEW

File: src/mbgl/renderer/render_orchestrator.hpp [hunk 1]
```diff
@@ -93,6 +93,7 @@ class RenderOrchestrator final : public GlyphManagerObserver, public ImageManage
                             const std::optional<std::string>& stateKey);
 
     void setTileCacheEnabled(bool);
+    bool getTileCacheEnabled();
```
Comment (louwers) [inline src/mbgl/renderer/render_orchestrator.hpp:96]: Can be const.

**Review by louwers — COMMENTED** [review]
Looks good, but we should probably have some kind of test coverage for this.

**Review by louwers — CHANGES_REQUESTED** [review]
Please add a simple test to the C++ unit tests (or modify an existing one) that just toggles it and renders something so we have some coverage. Ideally also for Android and iOS.

**Review by TimSylvester — APPROVED** [review]

**Review by louwers — APPROVED** [review]
Thanks for the smoke tests.

A test that tries to render without the cache enabled would be good at some point.

**Pull request merged**